config/reader.go

```diff
@@ -49,7 +49,7 @@ func loadConfigEnv(path string) (config *ProductConfig, err error) {
 		os.Exit(1)
 	}
 
-	if config.TFVars == "" || (config.TFVars != "k3s.tfvars" && config.TFVars != "rke2.tfvars") {
+	if config.TFVars == "" || (!strings.Contains(config.TFVars, "tfvars")) {
```

Comment:
we should keep the product validation

Reply:
This I added because I had multiple *.tfvars file with different settings so that I don't have to change the same file

Reply:
dont understood 

but why pushing it ?

Reply:
Is there anything wrong with pushing it?

Reply:
do we really want to restrict to these file names only?

Reply:
yeah because we are using only rke2 or k3s tfvars across everything

besides in this way if i pass  `test.tfvars` it will accept ...and it shouldnt if i dont have i file like this.

Reply:
we need to leave place for min error prone possible, especially on load

Reply:
Ok, updated